src/scss/components/ec-grid/ec-grid.story.js

```diff
@@ -1,176 +1,176 @@
-import { storiesOf } from '@storybook/vue';
```

Comment:
can we get ride of the file? I'm missing why we comment the whole thing out

Reply:
I have the same question here, for the rest looking good!

Reply:
Yeah I forgot to remove to remove this story. I removed the TF example because I think we should not have any reference to EBO and internal app logic in here.